boxoffice/templates/boxoffice.html

```diff
@@ -0,0 +1,619 @@
+{%raw%}
+<script src="https://checkout.razorpay.com/v1/checkout.js"></script>
+
+<div class="payment-progress-wrapper payment-stages-bg" id="payment-stages">
+  <ul class="progress-indicator">
+    {{#tabs:tab}}
+      <li class="indicator {{#if id === activeTab}}active{{/if}} {{#if complete}}completed{{/if}}">
+        <span class="bubble"></span>{{ label }}<br>
+      </li>
+    {{/}}
+  </ul>
+</div><!--eof payment-box-progress-wrapper -->
+<div class="payment-stages-wrapper">
+  {{#tabs:tab}}
+    {{#if id === 'boxoffice-selectItems' && id === activeTab}}
+      <div id='{{id}}' class="boxoffice-section ticket-selection-content clearfix" intro-outro='fly:{"x":0,"y":"0"}'>
+        {{#section.categories}}
+          <div id="{{name}}">
+            <div>
+              <h1 class="category-heading">{{title}}</h1>
+            </div>
+            <div>
+            {{#items}}{{# { item: . } }}
+              {{#order.line_items}} {{# { line_item: . } }}
+                {{#if line_item.item_name === item.name }}
+                  <div class="ticket-booking clearfix" id="{{line_item.item_name}}">
+                    <div class="ticket-details">
+                      <p class="ticket-title text-primary">{{ line_item.item_title }}</p>
+                      {{#line_item.discount_policies}}
+                        {{#if is_automatic || activated}}
+                          <p class="discount {{#activated}}applied{{/}}">
+                            {{ title }}{{#activated && !isLoadingFail}} <i class="fa fa-check"></i>{{/}}
+                          </p>
+                        {{/if}}
+                      {{/line_item.discount_policies}}
+                      <div class="ticket-description">
+                        <p class="valid-upto">Prices go up after {{line_item.price_valid_upto}}</p>
+                      </div>
+                      <div class="ticket-description">{{{ line_item.item_description }}}</div>
```

Comment:
Could add a Google Maps link to the venue details so that it opens up in Maps in a new tab on the phone for easy routing.